src/voice/connection.rs

```diff
@@ -291,6 +310,19 @@ impl Connection {
         packet: &[u8],
         nonce: &mut Nonce,
         ) -> Result<()> {
+        match demux::demux(packet) {
+            Demuxed::Rtp(rtp) => {},
+            Demuxed::Rtcp(rtcp) => {
+                println!("{:?}", rtcp);
```

Comment:
`debug!()` or `trace!()`

Same for next line.

Reply:
Will be replaced with actual logic to pass through to a `Receiver`, replacing the audio receiving system.